Library/Formula/nifi.rb

```diff
@@ -1,8 +1,8 @@
 class Nifi < Formula
   desc "Easy to use, powerful, and reliable system to process and distribute data."
   homepage "https://nifi.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=/nifi/0.4.0/nifi-0.4.0-bin.tar.gz"
-  sha256 "01226e19ac20ceea9e840889f8fa238289834c61380f7af5ec60a94b5c21348d"
+  url "https://www.apache.org/dyn/closer.cgi?path=/nifi/0.4.1/nifi-0.4.1-bin.tar.gz"
+  sha1 "b749c1ef5fbd0184d7aa6a864d3072e0ad576352"
```

Comment:
Feel free to file an upstream bug report asking for SHA256 checksums if you wish to. We have already in places, to varying degrees of success.